src/appleseed/renderer/kernel/rendering/final/uniformpixelrenderer.cpp

```diff
@@ -187,7 +185,10 @@ namespace
                             static_cast<float>(pt.y + s.y),
                             shading_result);
                     }
-                    else signal_invalid_sample();
+                    else
+                    {
+                        signal_invalid_sample();
```

Comment:
Remove the braces, leave the code as it was before.
Do the same in the other signal_invalid_sample() call a few lines after.